pbm/config.go

```diff
@@ -304,7 +329,7 @@ func (p *PBM) GetConfigVar(key string) (interface{}, error) {
 	case bson.TypeDouble:
 		return v.Double(), nil
 	case bson.TypeString:
-		return v.String(), nil
+		return v.StringValue(), nil
```

Comment:
I have changed this instruction from `v.String()` to `v.StringValue()` to avoid getting the quoted value back.
But there are probably side effects that I didn't see.

Reply:
@dAdAbird do you have a reason to keep using .String()?

Reply:
@dAdAbird  
Did you have the time to have a look at it, by any chance?

Reply:
Ping 😄